cdap-docs/admin-manual/build.sh

```diff
@@ -80,9 +82,16 @@ function download_includes() {
   types="configuration hdfs-permissions"
   for type in ${types}; do
     rewrite_references_in_place_sed "${target_includes_dir}/mapr-${type}.rst" " su hdfs" " su mapr"
-    rewrite_references_in_place_sed "${target_includes_dir}/mapr-${type}.rst" "FQDN1:2181,FQDN2:2181" "FQDN1:5181,FQDN2:5181"
   done
   
+  local source_pattern="(FQDN1:2181,FQDN2:2181)"
+  local target_pattern=$(cat <<EOF
+(FQDN1:5181,FQDN2:5181);
+             note that the MapR default of 5181 is different than the ZooKeeper default of 2181
```

Comment:
I agree this is an anti-pattern. However, given the time constraint, this was the fastest I could come up with. The reason for the problem is that this text is in the middle of a ~400-line file: https://github.com/caskdata/cdap/blob/docs/r4.1.0_8774_mapr/cdap-docs/admin-manual/source/_includes/installation/configuration.txt#L78 

So I don't really have access to that portion of the text unless I break the file. Or come up with a better way to handle this. These things are getting complicated, but we do want to reuse as much text as possible so that we keep everything in sync. I propose we let it go, and I'll file an issue to break these files and remove these changes from the `build.sh`.